lib/posthog/feature_flags.rb

```diff
@@ -56,7 +55,7 @@ def is_feature_enabled(key, distinct_id, default_result = false)
       else
         data = { 'distinct_id' => distinct_id }
         res = _request('POST', 'decide', false, data)
```

Comment:
make this `v=2`, instead of doing it in `_request` ?

Reply:
wym? `_request` turns it into `/decide/?v=2/?token` which is wrong :(  if I pass in `decide/?v=2` here

Reply:
Also do we really need the `get_feature_flag` call since `is_feature_enabled` essentially does the same thing? 😮

Reply:
it's grabbing the object value so it'll return the string of whatever variant it's at if it's multivariate for `is_feature_enabled`

Reply:
i mean though i guess technically "is_feature_enabled" would be incorrect if it's returning a string instead of a boolean... which means the python library needs fixing too... what do you think?

Reply:
Yep, python and this one both need `get_feature_flag`, as otherwise we have no way to know the variant in the response.

And like you say, `is_feature_enabled` is a boolean always.

Reply:
oh boy, that request implicit parsing is very annoying..

There's a `join` function to seamlessly merge params though: https://ruby-doc.org/stdlib-2.5.1/libdoc/uri/rdoc/URI/RFC2396_Parser.html#method-i-join-label-Args

Or better yet, you can change the `_request` signature to accept extra query params, which you can append at the end of the URI generation.

The way it is right now is sadly too confusing for whoever comes in the future :/